### PR TITLE
Added hide_context option to collection and tag admin

### PR DESCRIPTION
**Pull request by core23**
The ``CategoryAdmin`` already has this feature.

### CODE REVIEW

**Comment by OskarStark** [comment]
ping @rande

**Comment by core23** [comment]
Ping @rande @Soullivaneuh

**Comment by sagikazarmark** [comment]
:+1:

**Comment by OskarStark** [comment]
please check travis

**Comment by core23** [comment]
Can you restart travis @OskarStark? Sounds like a technical problem

**Comment by OskarStark** [comment]
Restarted...

**Comment by core23** [comment]
ping @OskarStark

File: Admin/ContextAwareAdmin.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+namespace Sonata\ClassificationBundle\Admin;
+
+use Sonata\AdminBundle\Admin\Admin;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\ClassificationBundle\Entity\ContextManager;
+
+abstract class ContextAwareAdmin extends Admin
+{
+    protected $contextManager;
```
Comment (OskarStark) [inline Admin/ContextAwareAdmin.php]: please add phpdoc

File: Admin/ContextAwareAdmin.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        return $instance;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configureDatagridFilters(DatagridMapper $datagridMapper)
+    {
+        $options = array();
+        if ($this->getPersistentParameter('hide_context') === 1) {
```
Comment (OskarStark) [inline Admin/ContextAwareAdmin.php]: please use YODA comaprison

File: Resources/config/admin.xml [hunk 3]
```diff
@@ -27,6 +27,7 @@
             <argument />
             <argument>%sonata.classification.admin.tag.entity%</argument>
             <argument>%sonata.classification.admin.tag.controller%</argument>
+            <argument type="service" id="sonata.classification.manager.context" />
```
Comment (OskarStark) [inline Resources/config/admin.xml]: we don't have variable for that? like the lines above?

**Comment by OskarStark** [comment]
except the comments, it looks good to me :+1:

File: Resources/config/admin.xml (hunk 3 above)
Comment (core23) [inline Resources/config/admin.xml]: The first parameters are strings, the fourth is an object. So we can't use any variable.

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**